test/controllers/AccountLinkControllerSpec.scala

```diff
@@ -191,8 +202,10 @@ class AccountLinkControllerSpec extends SpecBase {
         .thenReturn(Future.failed(new RuntimeException("Something went wrong")))
 
       running(app) {
-        val result = route(app, fakeRequest(
-          GET, controllers.routes.AccountLinkController.getAccountLinks(testSessionId).url)).value
+        val result = route(
+          app,
+          fakeRequest(GET, controllers.routes.AccountLinkController.getAccountLinks(testSessionId).url)
+        ).value
         status(result) mustBe INTERNAL_SERVER_ERROR
```

Comment:
There is a new line between this and the other line in the other running app sections, it might be worth mirroring it here for readability/consistency.

Reply:
done.